nixos/modules/services/networking/tinc.nix

```diff
@@ -279,6 +279,16 @@ in
               '';
             };
 
+            notify = mkOption {
+              default = false;
+              type = types.bool;
+              description = ''
+                Whether to set the system service type to "notify"; enable this if you want
+                to use system-notify in your tinc-up hook to tell systemd when the VPN interface
+                is up.
```

Comment:
Some example here would be nice like you did in the PR message:

```
${pkgs.systemd}/bin/systemd-notify --ready
```

Reply:
> Some example here would be nice like you did in the PR message:

Could you point me at how to do fully-fledged nix configuration examples in an option description with DocBook? I'd like to include the full `environment.etc`... example from https://github.com/NixOS/nixpkgs/pull/144118#issuecomment-956406822, since that's really what people will need to use this with, but I'm not sure how to make that appear right in the manual and the options search.